mongo/index_view.go

```diff
@@ -402,8 +402,9 @@ func (iv IndexView) drop(ctx context.Context, name string, _ ...*options.DropInd
 
 	selector := makePinnedSelector(sess, iv.coll.writeSelector)
 
-	op := operation.NewDropIndexes(name).
-		Session(sess).WriteConcern(wc).CommandMonitor(iv.coll.client.monitor).
+	// TODO(GODRIVER-3038): This operation should pass CSE to the DropIndexes
+	// Crypt setter to be applied to the operation.
```

Comment:
GODRIVER-3038 has been closed and this TODO has been resolved, so this comment can be removed.